src/box/lua/config/utils/schema.lua

```diff
@@ -1599,6 +1606,83 @@ end
 
 -- }}} <schema object>:pairs()
 
+-- {{{ <schema object>:jsonschema()
+
+-- This function sets common fields such as `description`, `default`, and `enum`
+-- for a given schema object. These fields are typical in JSON schemas.
+local function set_common_jsonschema_fields(res, schema)
+    res.description = schema.description
```

Comment:
Please add it to the list of annotations that are taken into account by the schema module. It is on the top of the module:

https://github.com/tarantool/tarantool/blob/8806b7b561d58d1cef79e6c9c723827f4ce67ed8/src/box/lua/config/utils/schema.lua#L3-L11

Reply:
Added.